### PR TITLE
Update to 45.0

### PR DESCRIPTION
**Pull request by mlundblad**
Update to libshumate 1.1.0, build with vector renderer support. Update libgweather to 4.4.0.

### CODE REVIEW

**Comment by A6GibKm** [comment]
The runtime was released a few days ago :)

**Comment by mlundblad** [comment]
> The runtime was released a few days ago :)

It still doesn't build, not sure why…

**Comment by A6GibKm** [comment]
It was not building because at the time the PR was made the 45 runtime didn't exist, it is also missing libportal. I hope you don't mind I  added it directly to the PR, this should trigger a rebuild. It builds locally for me with this.

File: org.gnome.Maps.json [hunk 1]
```diff
@@ -119,6 +119,23 @@
                 }
             ]
         },
+        {
```
Comment (mlundblad) [inline org.gnome.Maps.json:122]: Good catch! Missed that one…

**Comment by A6GibKm** [comment]
bot, build

**Pull request merged**